cpp/merge_print_and_online.cc

```diff
@@ -724,10 +724,16 @@ bool MergeFieldPair245(const MARC::Record::Field &merge_field, const MARC::Recor
             return true;
     }
 
-    const bool import_title_is_newer(import_record.getPublicationYear() > merge_record->getPublicationYear());
-    merge_record->insertField("246", { { 'a', import_title_is_newer ? import_title : merge_title },
-                                       { 'g', import_record.isElectronicResource() ? "electronic" : "print" + std::string(" title") } },
-                              /* indicator1 = */'2', /*indicator2 = */'3');
+    if (import_title_is_newer) {
+        *merge_field = import_field;
+        merge_record->insertField("246", { { 'a', merge_title },
```

Comment:
I thought you wanted to keep title + isElectronic of the newer title in this case?